docs/asyncapi.yaml

```diff
@@ -19,9 +19,11 @@ components:
       name: Crm Message
       contentType: application/json
       payload:
+        additionalProperties: false
         type: object
         properties:
           body:
+            additionalProperties: false
             type: object
             properties:
               sbi:
```

Comment:
For SBI and FRN coming in, I believe these are to be string -- dbl check that though

Reply:
Well the format for the example message I used to generate a statement seemed to use a number for SBI and FRN, so I assume not a string. As this is CRM though, I can't imagine the SBI and FRN numbers being classed as a string here when the SBI and FRN numbers in Payment Statement are classed as numbers

Reply:
Ah sorry, thought I left this under PaymentRequest's defintion

Just dbl check what is coming in -- it comes from `ffc-pay-statement-constructor` and I think a requirement was to send everything downstream as string

Reply:
I checked out the tests in ffc-pay-statement-constructor and it seemed that the mock FRN and SBI weren't stored as strings, so I guess it's okay for them to be numbers?

Reply:
I double checked in the code and both are numbers